tests/conftest.py

```diff
@@ -59,3 +60,12 @@ def base_plugins() -> LayeredConfigTree:
     }
 
     return LayeredConfigTree(config)
+
+
```

Comment:
This currently won't work without doing an editable install so we need to release this package and add to testing requirements before this can be merged.